### ISSUE TITLE
shape not matched

### ISSUE THREAD

**Issue by LeeKyungwook** [doantienthongbku/Implementation-patchnet#1]
Thanks for Great code. 

I found some errors. After inference network, shape is not matched with fc layer in loss. In loss func, fc layer need 1024 in features, but shape is not 1024. Can you check it?? Thx

**Comment by TangDL** [comment]
just change it to 512

**Comment by swhoosh** [comment]
@TangDL I also encountered this problem. Can you please tell me where can I change the feature size? I am very new to pytorch and NN. Thanks

**Comment by wlz987** [comment]
> Thanks for Great code.
> 
> I found some errors. After inference network, shape is not matched with fc layer in loss. In loss func, fc layer need 1024 in features, but shape is not 1024. Can you check it?? Thx
loss.py里面，把1024改成512
class PatchLoss(nn.Module):

    def __init__(self, alpha1=1.0, alpha2=1.0):
        super().__init__()
        self.alpha1 = alpha1
        self.alpha2 = alpha2
        self.sim_loss = SimilarityLoss()
        self.amsm_loss = AdMSoftmaxLoss(512, 2)
